libs/features/footer-promo.js

```diff
@@ -42,4 +44,6 @@ export default async function initFooterPromo(footerPromoTag, footerPromoType) {
   document.querySelector('main > div:last-of-type').insertAdjacentElement('afterend', section);
   await loadFragment(a);
   section.classList.add('section');
+  const sections = document.querySelectorAll('main > div');
+  decorateSectionAnalytics(section, sections.length - 1, config);
```

Comment:
Looking at `decorateSectionAnalytics` the 2nd param (`idx`) seems to indicate the section index, not the total amount of sections?

Reply:
where it's used normally, it's looping through all sections and passing in the array index from a querySelector all.  I don't want to rerun the other sections, but I am trying to determine what it's index would have been if it existed in time to be part of the initial loop.  And that number is 1 less than the length.